docs: improve documentation format (#2)
nearform-actions/k8s-kurated-addons-action: README.md

```yaml
- uses: nearform-actions/k8s-kurated-addons-action@main
  with:
    app-name: my-app-name
    app-port: 8080
    docker-registry: ghcr.io
    dockerfile-path: ./Dockerfile 
    kubernetes-manifest: ./knative.yaml
```

## Inputs

| input                 | required | default                        | description |
|-----------------------|----------|--------------------------------|-------------|
| `app-name`            | no       | `''`                           | The name of your app. This will be used both as the image name as well as the Knative service name for the deployment. |

> | `app-port`            | no       | `8080`                         | The port of your app. Usually this is the HTTP/HTTPS port your own app want to be approached to. |

Reviewer: ```suggestion
| `app-port`            | no       | `8080`                         | The port of your app. Usually this is the HTTP/HTTPS port your own app will be listening on. |
```
